docs/quickstart.adoc

```diff
@@ -159,7 +175,8 @@ module "oke" {
   ssh_public_key_path                   =   var.ssh_public_key_path
 
   label_prefix                          =   var.label_prefix
-  regions                               =   var.regions
+  home_region                           =   var.regions["home"]
```

Comment:
we don't have regions variable . Did I miss something?

Reply:
this will be in the user's own root and they'll need to define a variable regions

Reply:
or they can hardcode it